frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/utils/personnel-type-to-german-abbreviation.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { PersonnelType } from 'digital-fuesim-manv-shared';
+
+const personnelTypeToGermanAbbreviationDictionary: {
+    [Key in PersonnelType]: string;
+} = {
+    gf: 'GF',
+    notarzt: 'NA',
+    notSan: 'NFS',
+    rettSan: 'RS',
+    san: 'San',
+};
+
+@Pipe({
+    name: 'personnelTypeToGermanAbbreviation',
+})
+export class PersonnelTypeToGermanAbbreviationPipe implements PipeTransform {
```

Comment:
name: similar to above

Reply:
I named the pipes that output a german word from our internal logic that way to make it clear to non german speakers what they are for. In that case the internal logic is also in German what I think is a problem but renamig this would not solve the problem but introduce more